Silicon/TigerlakePkg/Library/TxtInit/TxtCtxLib.h

```diff
@@ -0,0 +1,300 @@
+/** @file
+  This file contains function definitions that can determine
+  the TXT capabilities of a platform during DXE and perform
```

Comment:
Seems confusing to leave references to DXE in this library. Should they be removed?

Reply:
Agree with @bejeanmo , there is no PEI and DXE concept in SBL. PEI and DXE should not be used in SBL code.
Better rename folder name TxtInit to TxtLib since all the files under this folder is for TxtLib.